README.rst

```diff
@@ -70,7 +70,7 @@ Containerized 'Production' environment
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    $ invenio-cli init --flavour=RDM
+    $ invenio-cli init rdm
```

Comment:
Probably you need to use `RDM` here, right?

Reply:
It has `case_sensitive=False`, so I think `rdm` should work.

Reply:
I personally prefer a lower-case version in the cli as it's one key less to press (shift). And yes, it's case insensitive.